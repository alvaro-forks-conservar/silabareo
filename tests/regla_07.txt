cabizbajo
acolitazgo
chozpa
satisfazte
mezquino
guzla
azre
lezna
majzén
bomba
dumdum
comto
digammas
columna
mazdeísmo
atrozmente
pechblenda
circumpolar
circumcirca