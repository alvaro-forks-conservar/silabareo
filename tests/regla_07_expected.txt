ca-biz-ba-jo
a-co-li-taz-go
choz-pa
sa-tis-faz-te
mez-qui-no
guz-la
az-re
lez-na
maj-zén
bom-ba
dum-dum
com-to
di-gam-mas
co-lum-na
maz-de-ís-mo
a-troz-men-te
pech-blen-da
cir-cum-po-lar
cir-cum-cir-ca